graphql_compiler/tests/schema_transformation_tests/test_rename_schema.py

```diff
@@ -183,6 +240,156 @@ def test_interface_rename(self):
             {"NewKid": "Kid", "NewCharacter": "Character"}, renamed_schema.reverse_name_map
         )
 
+    # def test_suppress_interface_implementation(self):
```

Comment:
Any reason this is commented? If not implemented, you can use `xfail` (search for examples in this repo).